src/ucs/sys/sock.h

```diff
@@ -320,7 +320,31 @@ int ucs_sockaddr_cmp(const struct sockaddr *sa1,
                      const struct sockaddr *sa2,
                      ucs_status_t *status_p);
 
+/**
+ * Indicate if given IP addr is INADDR_ANY (IPV4) or in6addr_any (IPV6)
+ * 
+ * @param [in]   addr       Pointer to sockaddr structure.
+ *
+ * @return 1 if input is INADDR_ANY or in6addr_any
+ *         0 if not
+ */
+int ucs_sockaddr_is_inaddr_any(struct sockaddr *addr);
 
+/**
+ * Return the number of devices and a string of names of devices under
+ * /sys/class/net that are in active state as determined by ucs_netif_is_active
+ *
+ * @param [out]    num_resources_p Return number of active net devices
+ * @param [in/out] dev_names_pp    Return pointer to string of device names
+ *                                 Length of string = num_resources_p * dev_name_len
+ *                                 Function allocates memory for the string and
+ *                                 user is responsible to free
+ * @param [in]     dev_name_len    Length of each of the device names
+ *
```

Comment:
pls add `@return`
something like `@return UCS_OK on success or ...` will be enough